plugins/media-keys/msd-media-keys-manager.c

```diff
@@ -590,7 +590,7 @@ do_touchpad_osd_action (MsdMediaKeysManager *manager, gboolean state)
 {
         dialog_init (manager);
         msd_media_keys_window_set_action_custom (MSD_MEDIA_KEYS_WINDOW (manager->priv->dialog),
-                                                 state ? "touchpad-enabled" : "touchpad-disabled",
```

Comment:
I added a symlink for symbolic icons too.
touchpad-enabled-symbolic.svg --> input-touchpad-symbolic.svg

Reply:
Other themes such as Adwaita use input-touchpad icon. if we use touchpad-enabled icon, touchpad-osd won't work when user changes theme from mate to another one. So I use input-touchpad instead of touchpad-enabled.

Reply:
got it.